src/app/main/component/events/components/event-editor/event-editor.component.ts

```
@@ -130,45 +170,118 @@
   }
 
   submitEvent(): void {
-    const formData = this.eventsService.prepareEventForSubmit(this.eventForm.value, this.eventId, this.isUpdating);
+    let sendEventDto = {
+      ...this.eventInformation.value,
+      datesLocations: this.eventDateForm.value.map((item) => {
+        if (!item.coordinates.latitude && !item.coordinates.longitude) {
+          delete item.coordinates;
+        }
+        if (!item.onlineLink) {
+          delete item.onlineLink;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)